app/templates/base.html

```diff
@@ -7,7 +7,7 @@
 {% block head %}
     <meta name="description" content="{{ config['SERVICE_NAME'] }}">
     <meta name="keywords"
-          content="GOV.UK, govuk, gov, government, uk, frontend, ui, user interface, jinja, python, flask, port, template, templating, macro, component, design system, html, forms, wtf, wtforms, widget, widgets, demo, example">
+          content="GOV.UK, govuk, gov, government, uk, frontend, ui, user interface, port, template, templating, component, design system, html, forms, widget, widgets, demo, example">
```

Comment:
Could we remove "demo", and "example" from here? I think they may have been from the initial flask setup. Maybe port too